sympy/printing/tests/test_tensorflow.py

```diff
@@ -129,6 +129,7 @@ def test_tensorflow_printing():
             " [tensorflow.math.exp(z), -t]])"
 
 
+@XFAIL
 def test_tensorflow_math():
```

Comment:
Later on someone might try to remove the XFAIL here if they can see the test passes. There should be a comment indicating why it is marked as XFAIL even though it usually passes. The comment should have a link to the issue.

Reply:
Ok sure. I'll add that. Thank you.

Reply:
Why are we testing random numbers? If we are just trying to see if the tensor flow result is the same as the result when evaluating the expr at the same value then why not just choose a non-pathological value like `-pi` to test for each of them?